tests/dns/dns-response-sticky-buffer/README.md

```diff
@@ -0,0 +1,4 @@
+Test the 'dns.response' sticky buffer.
+
+This test verifies that data in a name field or an rdata field 
+of a DNS response will trigger a signature.
```

Comment:
When you prepare a new version, could you please add the Redmine ticket reference, here?